src/App.js

```diff
@@ -8,7 +8,8 @@ function App() {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path='/user_access' element={<UserAccess />} />
+        <Route path="/user_access" element={<UserAccess />} />
+        <Route path="/home" element={<Home />} />
```

Comment:
Acho legal a ideia de ter as rotas no `Option.js` (acredito que `Routes.js` seria mais adequado. Mas aqui no Router poderia ter um map que colocaria todas essas rotas.
```suggestion
        <Route path="/user_access" element={<UserAccess />} />
        <UserRoute path="/home" element={<Home />} />
```